frontend/src/components/Comments/CommentsSection.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+type CommentsSectionProps = {
+  restaurantId: string;
+};
+
+function CommentsSection({ restaurantId }: CommentsSectionProps) {
+  const [comments, setComments] = useState<{ text: string }[]>([]);
+  const [newComment, setNewComment] = useState<{ text: string }>({ text: "" });
+
+  useEffect(() => {
+    axios
+      .get(`/api/restaurants/${restaurantId}/comments`)
+      .then((response) => {
+        setComments(response.data);
+      })
+      .catch((error) => {
+        console.error("There was an error fetching the comments!", error);
```

Comment:
Please log errors to betterstack. We can not see errors in the console of the users browser.